packages/admin/admin-sdk/src/config/types.ts

```diff
@@ -24,6 +24,17 @@ export interface RouteConfig {
    * An optional icon to display in the sidebar together with the label. If no label is provided, the icon will be ignored.
    */
   icon?: ComponentType
+
+  /**
+   * The nested route to display under existing route in the sidebar.
+   */
+  nested?:
+    | "/orders"
+    | "/products"
+    | "/inventory"
+    | "/customers"
+    | "/promotions"
+    | "/price-list"
```

Comment:
It would be good to move this to the admin-shared package both as a const: 

```
export const NESTED_ROUTE_POSITIONS = ["/orders", ...] as const
```
And as a type:
```
export type NestedRoutePosition = (typeof NESTED_ROUTE_POSITIONS)[number]
```

We should then use the const to validate that the user has passed a valid `nested` value in `generate-menu-items.ts`. And use the type in this file, and in the dashboard project.

Reply:
Really good point 🚀. Added type.